src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -49,17 +49,8 @@ public static string GetFullPath(string path)
             // unpredictable results.
             if (path.Contains('\0'))
```

Comment:
Initial issue is about this line.
The simplest fix would be splitting  Path.GetFullPath() on public (contains input checks) and internal (contains rest of the code).

Reply:
Ok. I'll switch the private to internal and make sure to remove any trailing whitespace on the two lines the checks pointed out. Thanks!

Reply:
I'm sorry, I mean there's a lot of GetFullPath() usings in the libraries. Of course you can stop on the initial fix.
For example, see GetVersionInfo(), methods in class Directory/File and many, many others where we could use the internal GetFullQualifiedPath().

Reply:
Ok. I'll submit a separate PR today, since these tests are almost done. If they approve the new refactoring, I'll apply it throughout the rest of the script.